### PR TITLE
Ionic user

### PR DESCRIPTION
**Pull request by Rachella**
Pop-up error messages for users when they input wrong info. All is working :+1:

### CODE REVIEW

File: www/js/controllers.js [hunk 1]
```diff
@@ -34,12 +34,22 @@ angular.module('starter.controllers', [])
       $scope.closeLogin();
     }).catch(function(response){
       console.log(response);
+      var message = '';
+      for (var i=0; i < response.errors.length; i++){
+        message += response.errors[i] + ' <br> ';
+      };
```
Comment (h8rry) [inline www/js/controllers.js]: You can use a popup template

Comment (h8rry) [inline www/js/controllers.js]: http://ionicframework.com/docs/api/service/$ionicPopup/

**Comment by h8rry** [comment]
@Rachella please review

File: www/js/controllers.js [hunk 2]
```diff
@@ -34,12 +34,22 @@ angular.module('starter.controllers', [])
...
+      $ionicPopup.alert({
+        title: 'Please Try Again',
+        template: 'Sorry, ' + message
```
Comment (Rachella) [inline www/js/controllers.js:40]:
``` js
  template: 'Sorry, ' + response.errors.join('<br>');
```

**Comment by Rachella** [comment]
Removed for loops and used join function in template.

**Pull request merged**